.github/workflows/build.yaml

```diff
@@ -6,10 +6,6 @@ concurrency:
   cancel-in-progress: true
 
 on:
```

Comment:
Changes to build.yaml should be reverted. When changes are made to the python script and pushed, a GitHub Action must be run automatically.